com.unity.toonshader/Documentation~/MatCap.md

```diff
@@ -50,70 +45,80 @@ MatCap color blending mode. Multiply or Add.
+
 
 ## Stabilize Camera Rolling
 Stabilize Camera rolling when capturing materials with camera.
 
-| Stabilize Camera Rolling Off  | Stabilize Camera Rolling On |
-| -- | --|
-| <img src="images/StabilizerOff.gif" height="256">| <img src="images/StabilizerOn.gif" height="256"> |
+<video title="A close-up on the face of a chibi-style character model. When the camera rotates, the specular highlights stay in the same place in the camera view." src="images/StabilizerOff.mp4" width="auto" height="auto" autoplay="true" loop="true" controls></video>
+Stabilize Camera Rolling disabled.
+
+<video title="The same close-up. When the camera rotates, the specular highlights stay in the same place on the hair." src="images/StabilizerOn.mp4" width="auto" height="auto" autoplay="true" loop="true" controls></video>
+Stabilize Camera Rolling enabled.
 
 
 ## Normal Map Specular Mask for MatCap
 If enabled, gives a normal map specifically for MatCap. If you are using MatCap as speculum lighting, you can use this to mask it.
 
-| Normal Map Specular Mask Off  | Normal Map Specular Mask On |
-| -- | --|
-| <img src="images/MatCapNormalMapOff.png" height="256">| <img src="images/MatCapNormalMapOn.png" height="256"> |
+![A view from above a chibi-style character model with blonde hair and brighter yellow specular highlights.](images/MatCapNormalMapOff.png)<br/>
+Normal Map Specular Mask disabled.
+
+![The same view. The individual hairs now have highlights.](images/MatCapNormalMapOn.png)<br/>
+Normal Map Specular Mask enabled.
 
 
 ### Normal Map
 A texture that dictates the bumpiness of the material.
 
-| Normal Map Texture Example | 
-| -- |
-| <img src="images/HairNormalMask.png" height="256">|
+
+![A grayscale texture of fine hair.](images/HairNormalMask.png)<br/>
+An example of a Normal Map texture.
+
 
 ### Rotate Normal Map UV
 Rotates the MatCap normal map UV based on its center.
 
-<img src="images/RotateMatCapNormalMapUV.gif" height="256">
+<video title="A view from above a chibi-style character model with blonde hair and brighter yellow specular highlights on individual hairs. The highlights rotate." src="images/RotateMatCapNormalMapUV.mp4" width="auto" height="auto" autoplay="true" loop="true" controls></video>
+
 
 ## MatCap Blending on Shadows
 Enables the blending rate of the MatCap range in shadows.
 
-| MatCap Blending on Shadows Off  | MatCap Blending on Shadows On |
-| -- | --|
-| <img src="images/MatCapBlendingOnShadowOff.png" height="256">| <img src="images/MatCapBlendingOnShadowOn.png" height="256"> |
+<canvas class="image-comparison" role="img" aria-label="A view from above a chibi-style character model with blonde hair. With MatCap Blending on Shadows on, the bright highlights on individual hairs are less visible.">
+    <img src="images/MatCapBlendingOnShadowOff.png" title="MatCap Blending on Shadows disabled.">
+    <img src="images/MatCapBlendingOnShadowOn.png" title="MapCap Blending on Shadows enabled.">
+</canvas>
```

Comment:
I checked [the result](https://yamato-artifactviewer.prd.cds.internal.unity3d.com/0ea85a0e-3f8b-490b-8317-0bd27b086420%2Fdocumentation%2FPackage_Doc/manual/index.html) with the content corresponding to [e62c5e9](https://github.com/Unity-Technologies/com.unity.toonshader/pull/424/commits/e62c5e927a4418b25b8a4f01b723dceb9bdc5301) the images are not shown here. I use Chrome.

Reply:
I think this is because the preview doesn't seem to use [Package Manager Doc Tools (PMDT)](https://docs.unity3d.com/Packages/com.unity.package-manager-doctools@3.10/manual/manual-xrefs.html), which has the functionality for sliders like this. 

The live site uses PMDT, so it should work fine. My local preview in Chrome looks ok:

![2025-06-05_08-44-00](https://github.com/user-attachments/assets/0ca3066f-88a3-4866-8fc2-d2d1bb9d8543)